lib/addons/external-dns/index.ts

```diff
@@ -22,9 +22,9 @@ const defaultProps = {
     name: 'external-dns',
     chart: 'external-dns',
     namespace: 'external-dns',
-    repository: 'https://charts.bitnami.com/bitnami',
+    repository: 'https://kubernetes-sigs.github.io/external-dns/',
     release: 'blueprints-addon-external-dns',
-    version: '6.20.3',
+    version: '1.12.0',
```

Comment:
is there a warning on deploy when using 1.12? There should be. 
Has it been confirmed to work with arm?

Reply:
@shapirov103 It did throw a warning which i ignored. I fixed it now and it works perfectly fine with ARM. Please run e2e and merge.